node-bb/README.md

```diff
@@ -0,0 +1,177 @@
+the list of Docker image versions on [https://hub.docker.com/_/mongo](https://hub.docker.com/_/mongo).
+
+#### Mac OSX and Windows
+
+This command will start the Docker container:  
+```shell script
+docker run --rm --name mongodb -p 27017:27017 -d mongo:4.4
+```
+
+To stop the container, run ```docker stop mongodb```
+
+The data saved will persist between shutdowns because it is being saved in the VirtualBox instance running on your os.
+For more information on that, see the section Where to Store Data at 
+[https://hub.docker.com/_/mongo](https://hub.docker.com/_/mongo).
+
+#### Linux and Unix
+
+While Linux and Unix can use the same command as Mac OSX to start the container, the data stored is not in VirtualBox.
+Instead, it is stored directly on the host at `/var/lib/docker/volumes/{volume hash}/_data`. It's far easier for backup
+purposes if you mount a volume directory for storing the data. The below command uses a volume mount.
+
+This command will start the Docker container:
+
+```shell script
+docker run --rm --name mongodb -p 27017:27017 -v /opt/triple-forums/datadir:/data/db -d mongo:4.4
+```
+
+To stop the container, run ```docker stop mongodb```
+
+The data saved is stored in the volume path you specify with the `-v` flag. In this case `/opt/triple-forums/datadir`.  
+
+### NodeBB
+
+The table below lists the version of NodeBB that TripleA Forums is currently running. This document 
+is written to that version.
+
+| TripleA Forums | NodeBB Version | Package Json                                                                              |
+|----------------|----------------|-------------------------------------------------------------------------------------------|
+| Current        | 1.13.3         | [package.json source](https://github.com/NodeBB/NodeBB/blob/v1.13.3/install/package.json) |
+| Future         | 1.14.3         | [package.json source](https://github.com/NodeBB/NodeBB/blob/v1.14.3/install/package.json) |
+
+#### Upgrading To A Newer NodeBB Version
+
+Upgrading to a newer version uses the following process:
+
+1. Change the version of the FROM image in the Dockerfile to the newer version of NodeBB.  
+2. Update the install/package.json to use packages from the newer versions package.json source.  
+3. Replace the nodebb-plugins in the install/package.json with the current list of plugins used for TripleA Forums.  
+4. Run Docker build to create the new image.
+
+#### Plugins  
+
+NodeBB customizations are plugins which are installed either through the admin interface or by adding them 
+to package.json directly. For the TripleA Forums, they are installed through the package.json.
+
+#### Installing/Upgrading A Plugin
+
+Installing or grading a plugin use the following process:
+
+1. Update the install/package.json to use the newer plugin.  
+2. Run Docker build to create the new image.
+
+##### List of currently installed plugins  
+
+NOTE: *Refer to install/package.json for the most up to date list.*
+
+```json
```

Comment:
Codacy found an issue: [[code-block-style] Code blocks should be indented](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6342386)